File: eBarberShop/eBarberShop.Model/Ocjene.cs
namespace eBarberShop.Model
{
    public class Ocjene
    {
        public int Id { get; set; }
        public DateTime Datum { get; set; }
        public double Ocjena { get; set; }
        public string? Opis { get; set; }
        public int UposlenikId { get; set; }
        public int KorisnikId { get; set; }

        public Korisnici Korisnik { get; set; }

        public string? korisnickoIme => Korisnik?.KorisnickoIme;
    }
}

File: eBarberShop/UI/ebarbershop_mobile/lib/models/ocjene/ocjene.dart
import 'package:json_annotation/json_annotation.dart';

part 'ocjene.g.dart';

@JsonSerializable()
class Ocjene {
  int id;
  DateTime datum;
  double ocjena;
  String? opis;
  int uposlenikId;
  int korisnikId;
  String? korisnickoIme;

  Ocjene(this.id, this.datum, this.ocjena, this.opis, this.uposlenikId, this.korisnikId, this.korisnickoIme);

  factory Ocjene.fromJson(Map<String, dynamic> json) =>
      _$OcjeneFromJson(json);

  Map<String, dynamic> toJson() => _$OcjeneToJson(this);
}